src/api/app/models/workflow/step/configure_repositories.rb

```diff
@@ -1,8 +1,6 @@
 class Workflow::Step::ConfigureRepositories < Workflow::Step
-  REQUIRED_KEYS = [:source_project, :repositories].freeze
```

Comment:
I don't see why we can't use a constant still. Every step needs to have this constant and if it doesn't, the validation is going to break. Nothing will work without this, so it's not possible to forget this constant.